src/components/HeroSection.vue

```diff
@@ -26,7 +26,7 @@
         <p>
           Search for free content in the public domain and under Creative Commons licenses.
           <br />
-          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-types-examples/">here</a>.
+          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-types-examples/" target="_blank">here</a>.
```

Comment:
Can you please add the `rel="noopener"` attribute in addition to `target="_blank"`?

For reference: https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
I didn't know about the 'rel' attribute. Thanks!